frontend/src/components/Shared/Map/CrossingMap.js

```diff
@@ -302,18 +239,18 @@ class CrossingMap extends React.Component {
         fitBounds={this.props.viewport}
         center={center}
       >
-        {!isLoading &&
-          showOpen && (
-            <Layer
-              type="symbol"
-              id="openCrossings"
-              layout={{ 'icon-image': 'open', 'icon-allow-overlap': true }}
-              filter={[
-                'all',
-                ['!=', 'crossingId', this.state.selectedCrossingId],
-              ]}
-            >
-              {openCrossings.map((crossing, i) => {
+        {showOpen && (
+          <Layer
+            type="symbol"
+            id="openCrossings"
+            layout={{ 'icon-image': 'open', 'icon-allow-overlap': true }}
+            filter={[
+              'all',
+              ['!=', 'crossingId', this.state.selectedCrossingId],
+            ]}
```

Comment:
Unrelated to this PR, but I wonder if we should do this filtering in the component instead of Mapbox world. When `selectedCrossingId` changes we have to re-render a  bunch of other stuff anyways, so we probably don't gett huge performance savings by letting Mapbox do the filtering.